drivers/gpio/gpio_stm32.c

```diff
@@ -194,37 +195,41 @@ static int gpio_stm32_init(struct device *device)
 }
 
 
-#define GPIO_DEVICE_INIT(__name, __suffix, __base_addr, __port, __cenr, __bus) \
-static const struct gpio_stm32_config gpio_stm32_cfg_## __suffix = {	\
-	.base = (u32_t *)__base_addr,				\
-	.port = __port,							\
-	.pclken = { .bus = __bus, .enr = __cenr }			\
-};									\
-static struct gpio_stm32_data gpio_stm32_data_## __suffix;		\
-DEVICE_AND_API_INIT(gpio_stm32_## __suffix,				\
-		    __name,						\
-		    gpio_stm32_init,					\
-		    &gpio_stm32_data_## __suffix,			\
-		    &gpio_stm32_cfg_## __suffix,			\
-		    POST_KERNEL,					\
-		    CONFIG_KERNEL_INIT_PRIORITY_DEFAULT,			\
+#define GPIO_DEVICE_INIT(__name, __suffix, __base_addr, __port, __cenr, __bus)		\
```

Comment:
What was the intent on changing these files?

Reply:
please see my second commit. I undid this unnecessary white space change.

Reply:
Ok, so please the fix the initial commit instead.